From beca90c378b55451f1d932f1e59fac06077700c9 Mon Sep 17 00:00:00 2001
From: Sam Hartman <hartmans@debian.org>
Date: Sat, 11 Jan 2014 17:29:18 -0500
Subject: .gbp.conf

Include pristine-tar configuration in .gbp.conf

Patch-Category: debian-local
---
 .gbp.conf | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 .gbp.conf

diff --git a/.gbp.conf b/.gbp.conf
new file mode 100644
index 0000000..6723fcc
--- /dev/null
+++ b/.gbp.conf
@@ -0,0 +1,2 @@
+[DEFAULT]
+pristine-tar=True
